src/data/drinks.ts

```diff
@@ -0,0 +1,348 @@
+export interface Drink {
```

Comment:
```suggestion
export typeDrink = {
```
Es basicamente lo mismo, pero para mi los props son interfaces y los tipos/objetos que se declaran son `type`